htmlreport/cppcheck-htmlreport

```diff
@@ -589,7 +610,11 @@ if __name__ == '__main__':
         output_file.write(HTML_HEAD_END.replace("content", "content_index", 1))
 
         output_file.write('\n       <table>')
-        output_file.write('\n       <tr><th>Line</th><th>Id</th><th>CWE</th><th>Severity</th><th>Message</th></tr>')
+        if add_author_information:
+            output_file.write('\n       <tr><th>Line</th><th>Id</th><th>CWE</th><th>Severity</th><th>Message</th><th></th><th>Author</th><th></th><th>Author mail</th><th></th><th>Date (DD/MM/YYYY)</th></tr>')
+        else:
+            output_file.write('\n       <tr><th>Line</th><th>Id</th><th>CWE</th><th>Severity</th><th>Message</th></tr>')
```

Comment:
how about adding some helper function for creating a `<tr>...</tr>` string. Something like:

    def tr_str(tr_class, td_th, line, id, cwe, severity, message, author=None, author_mail=None, date=None):
        ret = ''
        for item in (line, id, cwe, severity, message, author, author_mail, date):
            if item:
                ret += '<%s>%s</%s>' % (td_th, item, td_th)
        if tr_class:
            tr_attributes = ' class="%s"' % tr_class
        else:
            tr_attributes = ''
        return '<tr%s>%s</tr>' % (tr_attributes, ret)

Feel free to refactor my idea. Maybe it makes sense to pass `add_author_information` to this function also.